premake5.lua

```diff
@@ -132,7 +132,7 @@ elseif (os.istarget("windows")) then
     platforms { "x64", "x86" }
 
     filter "platforms:x86"
-        targetsuffix "32"
+        targetsuffix "_32bit_x86"
```

Comment:
why not just `Surge_x86.dll`? since `x86` basically means `32-bit`

Reply:
resolved